contracts/src/v0.8/ccip/test/pools/GHO/fork/GhoTokenPoolMigrate1_4To1_5/LegacyTestnetTokenPools/UpgradeableLockReleaseTokenPool_Sepolia.sol

```diff
@@ -0,0 +1,278 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import {ITypeAndVersion} from "../../../../../../../shared/interfaces/ITypeAndVersion.sol";
+
+import {ILiquidityContainer} from "../../../../../../../rebalancer/interfaces/ILiquidityContainer.sol";
+
+import {UpgradeableTokenPool} from "../../../../../../pools/GHO/UpgradeableTokenPool.sol";
+import {RateLimiter} from "../../../../../../libraries/RateLimiter.sol";
+
+import {IERC20} from "../../../../../../../vendor/openzeppelin-solidity/v4.8.3/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "../../../../../../../vendor/openzeppelin-solidity/v4.8.3/contracts/token/ERC20/utils/SafeERC20.sol";
+
+import {IRouter} from "../../../../../../interfaces/IRouter.sol";
+import {VersionedInitializable} from "./VersionedInitializable.sol";
+
+/// @title UpgradeableLockReleaseTokenPool_Sepolia
+/// @author Aave Labs
+/// @notice Upgradeable version of Chainlink's CCIP LockReleaseTokenPool
+/// @dev Contract adaptations:
+/// - Implementation of VersionedInitializable to allow upgrades
+/// - Move of allowlist and router definition to initialization stage
+/// - Addition of a bridge limit to regulate the maximum amount of tokens that can be transferred out (burned/locked)
+contract UpgradeableLockReleaseTokenPool_Sepolia is
+  VersionedInitializable,
+  UpgradeableTokenPool,
+  ILiquidityContainer,
+  ITypeAndVersion
+{
+  using SafeERC20 for IERC20;
+
+  error InsufficientLiquidity();
+  error LiquidityNotAccepted();
+  error Unauthorized(address caller);
+
+  error BridgeLimitExceeded(uint256 bridgeLimit);
+  error NotEnoughBridgedAmount();
+  event BridgeLimitUpdated(uint256 oldBridgeLimit, uint256 newBridgeLimit);
```

Comment:
Put this event before errors as per style guidelines: https://docs.soliditylang.org/en/v0.8.21/style-guide.html#order-of-layout

Reply:
(pls feel free to reopen if you feel we should add this)